packages/phishing-controller/src/PhishingDetector.ts

```diff
@@ -223,29 +224,48 @@ export class PhishingDetector {
    * @returns An object indicating if the URL is blocked and relevant metadata.
    */
   isMaliciousRequestDomain(urlString: string): PhishingDetectorResult {
```

Comment:
Can we rename this function? I understand that this function is just checking against the `c2DomainBlocklist` but I would imagine anything also on the `blocklist` is also considered malicious.